src/international-phone-number.coffee

```diff
@@ -73,7 +73,6 @@ angular.module("internationalPhoneNumber", []).directive 'internationalPhoneNumb
       if value
         validity = element.intlTelInput("isValidNumber")
         ctrl.$setValidity 'international-phone-number', validity
-        ctrl.$setValidity '', validity
       else
         value = ''
```

Comment:
Today I read into the docs regarding this matter. I found out that this is not the best way to do validation checking. Consider this:
```javascript
ctrl.$validators.internationalPhoneNumber = function(value) {
  return value && element.intlTelInput("isValidNumber");
};
```
This way it seems like a clean separation between parses (which prepare value for validation checking and eventually writing into model) and validation.